src/auro-checkbox-group.js

```diff
@@ -16,14 +24,13 @@
  * @attr {Boolean} noValidate - If set, disables auto-validation on blur.
  * @attr {Boolean} required - Populates the `required` attribute on the element. Used for client-side validation.
  * @attr {Boolean} horizontal - If set, checkboxes will be aligned horizontally.
+ * @slot legend - Allows for the legend to be overridden.
+ * @slot optionalLabel - Allows for the optional label to be overridden.
+ * @slot helpText - Allows for the helper text to be overridden.
+ * @fires auroCheckboxGroup-validated - Notifies that the `validity` value has changed.
+ * @fires auroCheckboxGroup-helpText - Notifies that the `setCustomValidity` value has changed.
```

Comment:
```
checkbox/src/auro-checkbox-group.js
  30:1  warning  The type 'auroCheckboxGroup-validated' is undefined  jsdoc/no-undefined-types
  31:1  warning  The type 'auroCheckboxGroup-helpText' is undefined   jsdoc/no-undefined-types
```

Where are these methods defined?

Reply:
Those are events, not methods, but those events are dispatched from the `auro-formvalidation` repo.